app/frontend/src/features/user/UserTextAndLabel.tsx

```diff
@@ -33,6 +34,7 @@ export const LabelsReferencesLastActive = ({ user }: Props) => (
           : LAST_ACTIVE_FALSE
       }
     />
+    <LabelAndText label={LOCAL_TIME} text={user.timezone} />
```

Comment:
if you create a file `src/utils/dayjs.ts` which is this:
```
import dayjs from "dayjs";
import Timezone from "dayjs/plugin/timezone";
import LocalizedFormat from "dayjs/plugin/timezone";
dayjs.extend(timezone).extend(LocalizedFormat);
export default dayjs;
```

and change all our dayjs imports to use that file, then I believe you should be able to do this:
```suggestion
    <LabelAndText label={LOCAL_TIME} text={dayjs().tz(user.timezone).format("LT")} />
```
And it will display the correct time in that timezone, in the current user's localized time display format.

Reply:
Awesome, thanks! I thought you or Darren would figure out how to do it really quickly so I wouldn't have to spend three hours on it :P